usaspending_api/references/tests/test_location.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from django.conf import settings
+from model_mommy import mommy
+import pytest
+
+from usaspending_api.common.api_request_utils import GeoCompleteHandler
+from usaspending_api.references.models import Location
+
+
+@pytest.mark.django_db
+def test_location_reference_fill():
+    country_code = mommy.make('references.RefCountryCode', country_code="USA", country_name="United States", _fill_optional=True)
+    city_county_code = mommy.make(
+        'references.RefCityCountyCode', city_code="A", county_code="B", _fill_optional=True)
+    location = mommy.make(
+        'references.Location', location_country_code=country_code, location_city_code="A", location_county_code="B")
+    assert location.location_city_name == city_county_code.city_name
+    assert location.location_county_name == city_county_code.county_name
+    assert location.location_state_code == city_county_code.state_code
+    assert location.location_country_name == country_code.country_name
+
+
+@pytest.mark.django_db
+def test_geocomplete_scope():
+    usa_code = mommy.make('references.RefCountryCode', country_code="USA", country_name="United States", _fill_optional=True)
+    can_code = mommy.make('references.RefCountryCode', country_code="CAN", country_name="Canada", _fill_optional=True)
+
+    usa_location = mommy.make(
+        'references.Location', location_country_code=usa_code)
+    can_location = mommy.make(
+        'references.Location', location_country_code=can_code)
+
+    no_scope_response = GeoCompleteHandler({"value": "a"}).build_response()
+    domestic_response = GeoCompleteHandler({"value": "a", "scope": "domestic"}).build_response()
+    foreign_response = GeoCompleteHandler({"value": "a", "scope": "foreign"}).build_response()
+    assert len(no_scope_response) == 2
+    assert domestic_response[0]["place"] == "United States"
+    assert len(domestic_response) == 1
+    assert foreign_response[0]["place"] == "Canada"
+    assert len(foreign_response) == 1
+
+
+@pytest.mark.django_db
+def test_geocomplete_limit():
```

Comment:
Same question here about the test location. Again, it's a minor point, and not something worth holding up a merge. The tests are great!